.github/workflows/request-reviewer.yml

```diff
@@ -9,13 +9,13 @@ on:
       - ready_for_review
       - reopened
       - synchronize
-    branches:
-      - 'main'
-      - 'FEATURE-BRANCH-*'
+  issue_comment:
+    types:
+      - created
 
 jobs:
   request-review:
-    if: github.event.pull_request.draft == false
+    if: github.event.pull_request && github.event.pull_request.draft == false && (github.event.sender.login == github.event.pull_request.user.login || github.event.action != 'created')
```

Comment:
just curious did we not need the null check `github.event.pull_request &&` before?

Reply:
theoretically, this workflow would trigger for comments on issues (in which case `github.event.issue` would be populated instead). But issues are disabled on this repo so it shouldn't ever happen. But I figure it doesn't hurt to be explicit?

(Previously this wasn't a question because pull_request_target only triggers for pull requests.)

Reply:
I see, yep it doesn't hurt. Thanks for the explanation